easycv/hooks/optimizer_hook.py

```diff
@@ -48,9 +46,6 @@ def __init__(self,
         self.multiply_key = multiply_key
         self.multiply_rate = multiply_rate
 
-    def before_run(self, runner):
```

Comment:
remove before_run zero_grad()，because original MMCV was not used. dino experiment is remove zero_grad() https://github.com/open-mmlab/mmcv/blob/master/mmcv/runner/hooks/optimizer.py